llvm/lib/Passes/StandardInstrumentations.cpp

```diff
@@ -720,9 +794,20 @@ void PrintIRInstrumentation::printBeforePass(StringRef PassID, Any IR) {
   if (!shouldPrintBeforePass(PassID))
     return;
 
-  dbgs() << "*** IR Dump Before " << PassID << " on " << getIRName(IR)
-         << " ***\n";
-  unwrapAndPrint(dbgs(), IR);
+  auto WriteIRToStream = [&](raw_ostream &Stream) {
+    Stream << "*** IR Dump Before " << PassID << " on " << getIRName(IR)
+           << " ***\n";
```

Comment:
If we are writing to stderr, or even more if we are writing an .ll file, it is convenient to be able to pipe the file directly into opt again. So how about we add a leading `; ` to make this header line a comment?

(Same goes for printAfter and printAfterPassInvalidated.)